022-quarkus-properties-config-all/src/main/java/io/quarkus/qe/configmapping/Person.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.qe.configmapping;
+
+import io.smallrye.config.ConfigMapping;
+
+@ConfigMapping(prefix = "person")
+public class Person {
+    String name;
+    int age;
+
+    public String getName() {
+        return name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+}
```

Comment:
getters can be deleted and a field can be accepted directly by calling a variable name. IMO this is the added value of config class vs. config interface

Reply:
Now I realize that the additional logic can be placed under getters. This is up to you now.

Reply:
Updated